bootstrap3 samples can't deploy to jboss as7.1

Wicket bootstratp3 samples fail to deploy to JBoss AS 7.1.1 Giving the following error with the weld service.

18:38:22,397 INFO  [org.jboss.as.controller](DeploymentScanner-threads - 1) JBAS014774: Service status report
JBAS014777:   Services which failed to start:      service jboss.deployment.unit."Veilig-Web.war".WeldService: org.jboss.msc.service.StartException in service jboss.deployment.unit."Veilig-Web.war".WeldService: com.google.common.collect.ComputationException: java.lang.ArrayIndexOutOfBoundsException: 3

18:42:12,469 INFO  [org.jboss.as.server.deployment](MSC service thread 1-3) JBAS015876: Starting deployment of "demo.war"
18:42:14,221 INFO  [org.jboss.weld.deployer](MSC service thread 1-4) JBAS016002: Processing weld deployment demo.war
18:42:14,230 INFO  [org.jboss.weld.deployer](MSC service thread 1-4) JBAS016005: Starting Services for CDI deployment: demo.war
18:42:14,245 INFO  [org.jboss.weld.deployer](MSC service thread 1-2) JBAS016008: Starting weld service for deployment demo.war
18:42:14,595 ERROR [org.jboss.msc.service.fail](MSC service thread 1-2) MSC00001: Failed to start service jboss.deployment.unit."demo.war".WeldService: org.jboss.msc.service.StartException in service jboss.deployment.unit."demo.war".WeldService: com.google.common.collect.ComputationException: java.lang.ArrayIndexOutOfBoundsException: 3
    at org.jboss.as.weld.services.WeldService.start(WeldService.java:83)
    at org.jboss.msc.service.ServiceControllerImpl$StartTask.startService(ServiceControllerImpl.java:1811) [jboss-msc-1.0.2.GA.jar:1.0.2.GA]
    at org.jboss.msc.service.ServiceControllerImpl$StartTask.run(ServiceControllerImpl.java:1746) [jboss-msc-1.0.2.GA.jar:1.0.2.GA]
    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145) [rt.jar:1.7.0_25]
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615) [rt.jar:1.7.0_25]
    at java.lang.Thread.run(Thread.java:724) [rt.jar:1.7.0_25]
Caused by: com.google.common.collect.ComputationException: java.lang.ArrayIndexOutOfBoundsException: 3
    at com.google.common.collect.ComputingConcurrentHashMap$ComputingMapAdapter.get(ComputingConcurrentHashMap.java:397)
    at org.jboss.weld.resources.ClassTransformer.loadClass(ClassTransformer.java:149)
    at org.jboss.weld.introspector.jlr.WeldClassImpl.<init>(WeldClassImpl.java:139)
    at org.jboss.weld.introspector.jlr.WeldClassImpl.of(WeldClassImpl.java:118)
    at org.jboss.weld.resources.ClassTransformer$TransformTypeToWeldClass.apply(ClassTransformer.java:49)
    at org.jboss.weld.resources.ClassTransformer$TransformTypeToWeldClass.apply(ClassTransformer.java:40)
    at com.google.common.collect.ComputingConcurrentHashMap$ComputingValueReference.compute(ComputingConcurrentHashMap.java:355)
    at com.google.common.collect.ComputingConcurrentHashMap$ComputingSegment.compute(ComputingConcurrentHashMap.java:184)
    at com.google.common.collect.ComputingConcurrentHashMap$ComputingSegment.getOrCompute(ComputingConcurrentHashMap.java:153)
    at com.google.common.collect.ComputingConcurrentHashMap.getOrCompute(ComputingConcurrentHashMap.java:69)
    at com.google.common.collect.ComputingConcurrentHashMap$ComputingMapAdapter.get(ComputingConcurrentHashMap.java:393)
    at org.jboss.weld.resources.ClassTransformer.loadClass(ClassTransformer.java:149)
    at org.jboss.weld.introspector.jlr.WeldClassImpl.<init>(WeldClassImpl.java:139)
    at org.jboss.weld.introspector.jlr.WeldClassImpl.of(WeldClassImpl.java:118)
    at org.jboss.weld.resources.ClassTransformer$TransformTypeToWeldClass.apply(ClassTransformer.java:49)
    at org.jboss.weld.resources.ClassTransformer$TransformTypeToWeldClass.apply(ClassTransformer.java:40)
    at com.google.common.collect.ComputingConcurrentHashMap$ComputingValueReference.compute(ComputingConcurrentHashMap.java:355)
    at com.google.common.collect.ComputingConcurrentHashMap$ComputingSegment.compute(ComputingConcurrentHashMap.java:184)
    at com.google.common.collect.ComputingConcurrentHashMap$ComputingSegment.getOrCompute(ComputingConcurrentHashMap.java:153)
    at com.google.common.collect.ComputingConcurrentHashMap.getOrCompute(ComputingConcurrentHashMap.java:69)
    at com.google.common.collect.ComputingConcurrentHashMap$ComputingMapAdapter.get(ComputingConcurrentHashMap.java:393)
    at org.jboss.weld.resources.ClassTransformer.loadClass(ClassTransformer.java:149)
    at org.jboss.weld.bootstrap.BeanDeployer.addClass(BeanDeployer.java:86)
    at org.jboss.weld.bootstrap.BeanDeployer.addClasses(BeanDeployer.java:115)
    at org.jboss.weld.bootstrap.BeanDeployment.createBeans(BeanDeployment.java:171)
    at org.jboss.weld.bootstrap.WeldBootstrap.deployBeans(WeldBootstrap.java:336)
    at org.jboss.as.weld.WeldContainer.start(WeldContainer.java:82)
    at org.jboss.as.weld.services.WeldService.start(WeldService.java:76)
    ... 5 more
Caused by: java.lang.ArrayIndexOutOfBoundsException: 3
    at org.jboss.weld.introspector.jlr.WeldConstructorImpl.<init>(WeldConstructorImpl.java:103)
    at org.jboss.weld.introspector.jlr.WeldConstructorImpl.of(WeldConstructorImpl.java:66)
    at org.jboss.weld.introspector.jlr.WeldClassImpl.<init>(WeldClassImpl.java:205)
    at org.jboss.weld.introspector.jlr.WeldClassImpl.of(WeldClassImpl.java:118)
    at org.jboss.weld.resources.ClassTransformer$TransformTypeToWeldClass.apply(ClassTransformer.java:49)
    at org.jboss.weld.resources.ClassTransformer$TransformTypeToWeldClass.apply(ClassTransformer.java:40)
    at com.google.common.collect.ComputingConcurrentHashMap$ComputingValueReference.compute(ComputingConcurrentHashMap.java:355)
    at com.google.common.collect.ComputingConcurrentHashMap$ComputingSegment.compute(ComputingConcurrentHashMap.java:184)
    at com.google.common.collect.ComputingConcurrentHashMap$ComputingSegment.getOrCompute(ComputingConcurrentHashMap.java:153)
    at com.google.common.collect.ComputingConcurrentHashMap.getOrCompute(ComputingConcurrentHashMap.java:69)
    at com.google.common.collect.ComputingConcurrentHashMap$ComputingMapAdapter.get(ComputingConcurrentHashMap.java:393)
    ... 32 more

18:42:14,812 INFO  [org.jboss.as.server](DeploymentScanner-threads - 2) JBAS015870: Deploy of deployment "demo.war" was rolled back with failure message {"JBAS014671: Failed services" => {"jboss.deployment.unit.\"demo.war\".WeldService" => "org.jboss.msc.service.StartException in service jboss.deployment.unit.\"demo.war\".WeldService: com.google.common.collect.ComputationException: java.lang.ArrayIndexOutOfBoundsException: 3"}}
18:42:14,813 INFO  [org.jboss.as.controller](DeploymentScanner-threads - 2) JBAS014774: Service status report
JBAS014777:   Services which failed to start:      service jboss.deployment.unit."Veilig-Web.war".WeldService: org.jboss.msc.service.StartException in service jboss.deployment.unit."Veilig-Web.war".WeldService: com.google.common.collect.ComputationException: java.lang.ArrayIndexOutOfBoundsException: 3
      service jboss.deployment.unit."demo.war".WeldService: org.jboss.msc.service.StartException in service jboss.deployment.unit."demo.war".WeldService: com.google.common.collect.ComputationException: java.lang.ArrayIndexOutOfBoundsException: 3

18:42:14,815 ERROR [org.jboss.as.server.deployment.scanner](DeploymentScanner-threads - 1) {"JBAS014653: Composite operation failed and was rolled back. Steps that failed:" => {"Operation step-2" => {"JBAS014671: Failed services" => {"jboss.deployment.unit.\"demo.war\".WeldService" => "org.jboss.msc.service.StartException in service jboss.deployment.unit.\"demo.war\".WeldService: com.google.common.collect.ComputationException: java.lang.ArrayIndexOutOfBoundsException: 3"}}}}
18:42:14,853 INFO  [org.jboss.as.server.deployment](MSC service thread 1-2) JBAS015877: Stopped deployment demo.war in 38ms

I suspect this is to do with google guava.

I will try on glassfish 4 and wildfly.

--- a/pom.xml
+++ b/pom.xml
@@ -4,7 +4,7 @@
     <parent>
         <groupId>de.agilecoders.maven</groupId>
         <artifactId>maven-parent-pom</artifactId>
-        <version>0.3.0</version>
+        <version>0.3.3</version>
     </parent>
 
     <groupId>de.agilecoders.wicket</groupId>
@@ -40,8 +40,8 @@
         <github.global.server>github</github.global.server>
 
         <bootstrap.version>3.0.0</bootstrap.version>
-        <config.version>0.3.0</config.version>
-        <wicket-webjars.version>0.3.0</wicket-webjars.version>
+        <config.version>0.3.3</config.version>
+        <wicket-webjars.version>0.3.1</wicket-webjars.version>
 
         <enforcer.disable>true</enforcer.disable>
         <pmd.disable>true</pmd.disable>

